### PR TITLE
Implement Continue the Momentum Section

### LINKED ISSUE #215: Continue the Momentum Section
## Description

Implement the Continue the Momentum Section and List, and integrate into contentful.

Image

## Todo

- [ ] Create Continue the Momentum section
- [ ] Add Continue the Momentum to nav bar
- [ ] Integrate into contentful

### PR DESCRIPTION
**Pull request by kennykos**
## Summary

Closes #215 

## Screenshots

image

### CODE REVIEW

File: src/sections/take-action/momentum.tsx [hunk 1]
```diff
@@ -1,13 +1,32 @@
...
 
-const Momentum = () => {
+import { HighlightedTitle, ListItem } from '../../components'
+import type { ListEntryDocument } from '../../types'
+import { mainParagraphStyle } from '../../utils/renderer'
+
+const Momentum: FC<{ entries: ListEntryDocument[] }> = ({ entries }) => {
   return (
-    <div>
+    <div className="px-12">
```
Comment (lizageorge) [inline src/sections/take-action/momentum.tsx:10]: The page needs some whitespace on the bottom, i'd recommend adding a `pb-36` here. Also the side padding is less than that on the figma, maybe use `px-16` instead.

File: src/sections/take-action/momentum.tsx [hunk 2]
```diff
@@ -1,13 +1,32 @@
...
       <HighlightedTitle
         title="Continue the Momentum"
         size="large"
         color="clementine"
       />
+      <div className="my-6 w-full md:px-12">
```
Comment (lizageorge) [inline src/sections/take-action/momentum.tsx:16]: There needs to be signaficantly more white space between header and list, maybe `mt-16` instead.

File: src/utils/content.ts [hunk 3]
```diff
@@ -178,6 +181,8 @@ export class ContentWrapper {
     switch (entity) {
       case 'takeActionPage':
         return entries[0] as TakeActionPage
+      case 'continueTheMomentumEntries':
```
Comment (lizageorge) [inline src/utils/content.ts:184]: More so a comment on the Contentful content - the Figma had all the list items ending in a comma, except the last one which ends in an exclamation point.

**Review by lizageorge — DISMISSED** [review]

File: src/sections/take-action/momentum.tsx [hunk 4]
```diff
@@ -1,13 +1,32 @@
...
+        <div className="ml-[3%] space-y-6">
```
[the remark this answers is not in the document]
Comment (kennykos) [reply src/sections/take-action/momentum.tsx]: Addressed in commit [de0cd59bc80c5899054ab2a53d186dbf1c4476cf](https://github.com/toriis-portal/toriis/pull/217/commits/de0cd59bc80c5899054ab2a53d186dbf1c4476cf)

File: src/utils/content.ts (hunk 3 above)
Comment (kennykos) [reply src/utils/content.ts:184]: Changed via contentful!

File: src/sections/take-action/momentum.tsx [hunk 5]
```diff
@@ -1,13 +1,32 @@
...
   return (
-    <div>
+    <div className="px-16 pb-36">
       <HighlightedTitle
         title="Continue the Momentum"
         size="large"
         color="clementine"
       />
+      <div className="my-6 mt-16 w-full md:px-12">
+        <div className="ml-[3%] w-4/6 space-y-6">
```
Comment (zorazrr) [inline src/sections/take-action/momentum.tsx:17]: For mobile responsiveness maybe add some media breakpoints here.

**Review by zorazrr — CHANGES_REQUESTED** [review]

File: src/sections/take-action/momentum.tsx (hunk 5 above)
Comment (kennykos) [reply src/sections/take-action/momentum.tsx:17]: It seem's to be pretty responsive just with the `md:px-12` already hear (also, I took this formatting from the Institutional Divestment section on the Home page)
Here is what the mobile view looks like for me
image

Comment (zorazrr) [reply src/sections/take-action/momentum.tsx:17]: on iPhone XR it's kinda squeezed together
image

Comment (kennykos) [reply src/sections/take-action/momentum.tsx:17]: That's so odd, on my version of iPhone XR it looks fine
image

Comment (kennykos) [reply src/sections/take-action/momentum.tsx:17]: Wait sorry I forgot to push my commit :hand_over_mouth:

Comment (kennykos) [reply src/sections/take-action/momentum.tsx]: ok should be good now

**Review by zorazrr — APPROVED** [review]
lgtm

**Pull request merged**